preprocessing.py

```diff
@@ -41,18 +42,21 @@ def parser():
 
 def is_image(file):
     _, ext = os.path.splitext(file)
-    return ext.lower() in {'.jpg', '.jpeg', '.png'}
+    return ext.lower() in {'.jpg', '.jpeg', '.png', '.bmp'}
 
 
 def run(input_folder, output_folder, recursive):
     paths = glob(input_folder + '/**/*.*',
                  recursive=True) if recursive else glob(input_folder + '/*')
     paths = filter(is_image, paths)
     meta = defaultdict(list)
-    for path in paths:
+    for path in tqdm(paths):
         try:
             face, rotation, face_found, original_res = \
                 functions.detectFace(path)
+            if os.path.splitext(path)[-1] == '.bmp':
```

Comment:
Misschien wel nog goed om er ook als comment even bij te zetten dat 'echt' converteren hier niet nodig is.